README.md

```diff
@@ -1,3 +1,5 @@
+[![Puppet Forge](http://img.shields.io/puppetforge/v/katello/gutterball.svg)](https://forge.puppetlabs.com/katello/gutterball)
```

Comment:
It doesn't matter, github proxies the image through HTTPS.  If you view the actual page's source you'll see it hidden by somehing like https://camo.githubusercontent.com/5f221b9bdad85b7e7d031ee2097d287f819a17c2/687474703a2f2f696d672e736869656c64732e696f2f707570706574666f7267652f762f6b6174656c6c6f2f67757474657262616c6c2e737667

If you can't see it, it might be the service, they were up and down all day yesterday.